terraform/azure/modules/hana_node/main.tf

```diff
@@ -153,11 +153,11 @@ resource "azurerm_lb_rule" "hanadb_exporter" {
 # hana network configuration
 
 resource "azurerm_network_interface" "hana" {
-  count                         = var.hana_count
-  name                          = "nic-${var.name}${format("%02d", count.index + 1)}"
-  location                      = var.az_region
-  resource_group_name           = var.resource_group_name
-  enable_accelerated_networking = var.enable_accelerated_networking
+  count                          = var.hana_count
+  name                           = "nic-${var.name}${format("%02d", count.index + 1)}"
+  location                       = var.az_region
+  resource_group_name            = var.resource_group_name
+  accelerated_networking_enabled = var.enable_accelerated_networking
```

Comment:
They renamed the arg in the provider? What a silly change, which must have probably broken a lot of customer configurations :D